film_list.sql

```diff
@@ -0,0 +1,16 @@
+SELECT COUNT(fid) FROM film_list;
+
+SELECT title, description FROM film_list ORDER BY description DESC LIMIT 10;
+
+SELECT title, length FROM film_list ORDER BY length LIMIT 1; 
+
+SELECT title, description FROM film_list WHERE actors LIKE '%CAMERON STREEP%';
+
+SELECT DISTINCT category AS "Genre", avg(price) AS "Price", avg(length) AS "Length" FROM film_list GROUP BY category;
+
+SELECT DISTINCT rating AS "Rating", COUNT(rating) AS "Count" FROM film_list GROUP BY rating;
```

Comment:
```suggestion
SELECT category AS "Genre", avg(price) AS "Price", avg(length) AS "Length" FROM film_list GROUP BY category;

SELECT rating AS "Rating", COUNT(rating) AS "Count" FROM film_list GROUP BY rating;
```

You can leave out the `DISTINCT` modifier when you are using `GROUP BY`. `GROUP BY` will make groups based on the distinct values in that column.